app/src/main/java/com/gdgevents/gdgeventsapp/features/event/presentaion/home/HomeScreen.kt

```diff
@@ -1,11 +1,123 @@
 package com.gdgevents.gdgeventsapp.features.event.presentaion.home
 
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
-import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.gdgevents.gdgeventsapp.features.event.data.Events
+import com.gdgevents.gdgeventsapp.features.event.presentaion.home.composables.EventListSection
+import com.gdgevents.gdgeventsapp.features.event.presentaion.home.composables.FeaturedEventsSection
+import com.gdgevents.gdgeventsapp.features.event.presentaion.home.composables.TopBar
+import com.gdgevents.gdgeventsapp.R
 
 @Composable
 fun HomeScreen(
-    viewModel: HomeViewModel = hiltViewModel(),
+    featuredEvents: List<Int>,
+    modifier: Modifier = Modifier,
 ) {
-    // add ui
+    LazyColumn(modifier = modifier.padding(16.dp)) {
+        item {
+            TopBar()
+        }
+        item {
+            FeaturedEventsSection(featuredEvents)
+        }
+        item {
+            EventListSection(
+                title = "Upcoming Events",
+                events = listOf(
```

Comment:
As this a dimmy data, put it in the data class so we can remove it once from there.